inlang/documentation/ecosystem/overview/architecture.md

```diff
@@ -1,29 +1,19 @@
 # Architecture
 
-Inlang's architecture consists of four main components:
+Inlang's architecture consists of three main components:
 
-1. [Applications](/documentation/concept/app) - Provide users with an interface for their projects.
-2. [SDK](/documentation) - Can be used by developers to build inlang apps.
-3. [Project file](/documentation/concept/project) - Defines the project's settings, which language tags exists and to be imported modules.
-4. [Modules](/documentation/plugin) - Contain plugins and/or lint rules that define the behavior of a project.
+1. [inlang Apps](/documentation/concept/app) - Provide users with an interface for their projects.
+2. [SDK](/documentation/sdk) - Can be used by developers to build inlang apps.
+3. [Repository (Lix)](/#lix) - Defines the project's settings, which language tags exists and to be imported modules.
```

Comment:
3 doesn't feel right to me yet - how about

> Lix provides a file-system and git-repo versioning used by all inlang apps, even those running in a browser.

Reply:
> Lix provides file-system and git-style versioning for all inlang apps, in browser and desktop environments.

Reply:
This is indeed incorrect. Lix does not define project settings. 

"Repository - The place where inlang projects are saved, and the system that provides inlang projects with change control."